nemoguardrails/eval/evaluate_hallucination.py

```diff
@@ -100,7 +104,10 @@ def check_hallucination(self):
 
         for question in tqdm.tqdm(self.dataset):
             with llm_params(self.llm, temperature=0.2, max_tokens=100):
-                bot_response = self.llm(question)
+                try:
+                    bot_response = self.llm(question)
+                except ValueError as e:
+                    bot_response = ""
```

Comment:
Agree with @trebedea - these should not be _silent_ errors. 

If an error occurs here, either: 
1. There is some issue with the LLM deployment. In this case, we should be solving before proceeding (for instance, if max_tokens is set to lower than required in the parameters even though the model is capable of longer generations), or
2. There is some very specific error condition (for instance, the prompt length is larger than the max possible context length of the underlying model itself). In this case, we should be catching the exact specific error only instead of a generic `ValueError`. Also, we should ideally return this error, and store it in a new column of the results dataframe if the output of the evaluation script is a dataframe (or at least print it using `log.error` or `log.warning`).